calSprite.py

```diff
@@ -304,7 +312,9 @@ async def notice(self, text):
         msg = text[text.find(parameters[1][1:]) :]
 
         if msg.upper().startswith("REPORT"):
-            await self.send(f"PRIVMSG #reports {nick}: {msg}")
+            currTime = datetime.datetime.now
+            await self.send(f"PRIVMSG #reports {nick}: {msg} - at ",  )
```

Comment:
the follow up commit fixes the fact that i completely forgot to finish this part